os/include/tinyara/sched.h

```diff
@@ -550,6 +550,9 @@ struct tcb_s {
 	sq_queue_t sigpendactionq;	/* List of pending signal actions      */
 	sq_queue_t sigpostedq;		/* List of posted signals              */
 	siginfo_t sigunbinfo;		/* Signal info when task unblocked     */
+#ifdef CONFIG_SIGKILL_HANDLER
+	_sa_sigaction_t usr_kill_handler;
```

Comment:
1. Please leave a comment to know what it is like others.
2. Usually starting with `_` means internal only. Is there any other signal handler? Does it use `_xxx_t` as well?
3. Let's follow naming rules for sig like above.
   i.e. `sigkillusrhandler`